overlord/devicestate/devicectx.go

```diff
@@ -30,11 +32,28 @@ func DeviceCtx(st *state.State, task *state.Task, providedDeviceCtx snapstate.De
 	if providedDeviceCtx != nil {
 		return providedDeviceCtx, nil
 	}
-	modelAs, err := findModel(st)
+
+	// see if we have a remodel in progress
+	if task != nil {
+		var modelass string
```

Comment:
Sorry for nitpicking, I meant this name as well :)

Reply:
Ups, missed this one. Fixed too.